samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
```

Comment:
would you mind providing a bit more documentation here, particularly for users who may not be familiar with the registy format?